.github/workflows/lint.yml

```diff
@@ -0,0 +1,52 @@
+name: Lint
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-versions: [12.x]
```

Comment:
Can this work with `16.x`?

Reply:
Should be. Locally I am using version 18 without problems.

But I have not fixed any findings so far

Reply:
hmm.. I had issues with 16 and 18 🤔 

I was able to run `npm install` with `v14.14.0`

Reply:
got the branch working in my personal repo with node 16 and passing CI except for the changelog check:
https://github.com/devlinjunker/news/pull/1

I can create a PR to this branch once it is cleaned up a little more, gonna be away this weekend though so not sure when I'll get around to it exactly